docs/scanners/slither.md

```diff
@@ -0,0 +1,8 @@
+# Slither Scanner
+
+Performs static analysis on solidity code.
+Only projects that use truffle or hardhat configs are supported.
+
+## Configuration
+
+Placeholder for future config options.
```

Comment:
We can add more details in this doc in the follow-up pr. Thank you for adding this placeholder doc.